tox.ini

```diff
@@ -35,13 +35,15 @@ ignore = E501,E265,E241,E704,E251,W504,E231,W503,E203
 [testenv:docs]
 deps =
     --requirement={toxinidir}/requirements-docs.txt
+    --requirement={toxinidir}/requirements-minimal.txt
```

Comment:
requirements-minimal.txt is not meant to be installed - if we need a real venv, we probably want to install from requirements.txt instead

and just curious: what error is this fixing?

Reply:
https://github.com/Yelp/Tron/actions/runs/9070754496/job/24923133131#step:7:333